### ISSUE TITLE
exchange calendar 2013 is read only

### ISSUE THREAD

**Issue by orclex** [Ericsson/exchangecalendar#468]
Hi,

I have installed thunderbird 45.0 and the exchangecalendar 3.7 addon. When configuring a new MS Exchange 2013 network calendar I enter the Server URL, email address, username and domain name. With previous installations I clicked on "check server and mailbox" and could select my calendar folder. After that the configuration was completed and I had access on my exchange calendar.

Now I click on "check server and mailbox" and get the message:

**Only user availability status for the mailbox's calendar will be visible.**

I can complete the configuration but my exchange calendar is read only. It is not possible to enable write access. I can disable the check box _read only_ in the calendar properties but it still remains read only.
I'm not sure if this is a bug in the exchangecalendar addon or a configuration problem with exchange 2013. But I know that it was working before and we had no change in the configuration on the exchange server except security updates. 

Maybe someone knows the reason for that problem?
Any ideas would be very appreciated!

Thanks you very much,
Alexander

**Comment by bavincen** [comment]
can you check the permissions for that calendar in folder properties.

**Comment by orclex** [comment]
Permissions are set to standard. Otherwise MS Outlook 2010-2016 and Outlook Web Access would not work too. I added for a test full permissions to a calendar folder but it is still only readable for the thunderbird exchangecalendar addon.

**Comment by michael-higgins** [comment]
I have this same problem with the latest beta version. If I uncheck readonly I still cannot add events. Most events have (busy) added after them. The calendar work in Outlook web version.

**Comment by DdRanna** [comment]
I have the very same problem. The calendar is read only although permissions are standard.
If I uncheck read only nothing changes. I can't even see free/busy stats...

**Comment by Timberwolf76275** [comment]
I also have the problem I have tried everything to fix this. Various xpi's from 3.3.1 to 3.7 also this

1. Removed all extensions from Thunderbird
2. sudo apt-get purge thunderbird*
3. rm -rf ~/.thunderbird/
4. rm -rf ~/.cache/thunderbird/
5. Reinstall & reconfigure mail, calendar

When I reinstalled TB and setup the calendar it is working untill I do the big restart or close Thunderbird when I open it again the calendar is read only
I am currently using ver. 38.8.0 thunderbird and exchange plugin 3.7.0

**Comment by knowrob** [comment]
Same problem here, only see busy status. It has worked fine for a long while, but with one of the recent updates stopped showing calendar events for one of my calendars. For another one, it still works.

45.2.0+build1-0ubuntu0.14.04.3
exchangecalendar version 3.8.0

**Comment by bavincen** [comment]
All of you are using more than one exchange calendar with TB please confirm?

**Comment by Timberwolf76275** [comment]
Hi
my IT. Dept found the problem you cannot use the password manager in t-bird there is a bug the best way is what is mentioned above.

    Removed all extensions from Thunderbird
    sudo apt-get purge thunderbird*
    rm -rf ~/.thunderbird/
    rm -rf ~/.cache/thunderbird/
    Reinstall & reconfigure mail, calendar

but when you reinstall do not save password with password manager for the email or the exchange calendar after I did this it has been working fine for over 1 month

**Comment by knowrob** [comment]
@bavincen Yes, I am using two Exchange calendars and several Google calendars

**Comment by bavincen** [comment]
can we add it to wiki @Timberwolf76275

**Comment by bavincen** [comment]
@Timberwolf76275 this resolves issue #464 ?

**Comment by Timberwolf76275** [comment]
go ahead and add it

**Comment by mtenorth** [comment]
Any news on this issue? Not using the password manager seems to be  a workaround, but it's not really a solution.

**Comment by orclex** [comment]
Unfortunately the error still exists in exchangecalendar v3.9.0 with Thunderbird 52.0.1

**Comment by Timberwolf76275** [comment]
The bug is still there do not save your password for the calendar and exchange works perfectly
